lib/configuration/llm_dependency_validator.rb

```diff
@@ -10,16 +10,9 @@ def initialize(opts = {})
       def valid_value?(val)
         return true if val == "f"
 
-        if @opts[:name] == :ai_summarization_enabled || @opts[:name] == :ai_helper_enabled
-          has_llms = LlmModel.count > 0
-          @no_llms_configured = !has_llms
-          has_llms
-        else
-          @llm_dependency_setting_name =
-            DiscourseAi::Configuration::LlmValidator.new.choose_llm_setting_for(@opts[:name])
+        @llm_dependency_setting_name = :ai_default_llm_model
 
-          SiteSetting.public_send(@llm_dependency_setting_name).present?
-        end
+        SiteSetting.public_send(@llm_dependency_setting_name).present?
```

Comment:
This could be an `OR` and check if the assigned persona has a `default_llm_id` I think.